app/assets/stylesheets/publisher.scss

```diff
@@ -18,7 +18,7 @@
       margin-top: 0;
     }
 
-    #publisher_textarea_wrapper { border: 1px solid $border-grey !important; }
+    #publisher_textarea_wrapper { border: 1px solid #ccc !important; }
```

Comment:
- Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Avoid using id selectors
- !important should not be used
- Selector `publisher_textarea_wrapper` should be written in lowercase with hyphens